### DISCUSSION TITLE
My Experience Being Locked Out For Not Enabling 2FA

### DISCUSSION THREAD

**Discussion by kuroodo** [community/community#74447] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

So I was locked out of my GitHub account for a few weeks because I refused to enable 2FA.

Suddenly I get an email stating that I will soon be required to enable 2FA. I log in to GitHub and for some reason I have access to my account again haha. Now there's a new deadline for me to enable 2FA (which I refuse unless they let us use email for 2FA).

Here's some notes from my experience:

### You Will Be Barred From Accessing GitHub In General

Once you get the screen that says to enable 2FA, and after the 7 day extra grace period, you will no longer have access to GitHub.

Until you enable 2FA you will be unable to:

- Log out of your account
- Contact support
- Look at any repository
- Do anything anywhere on GitHub

_I could not find a way to go past the screen requiring I enable 2FA. If you know of a way, please do share._

As a result, you will have to clear out your browsers cache, use an incognito tab, or use a different browser in order to access GitHub.
This means that **you will be unable to cancel any subscriptions and delete any personal information from your account**. I made a new account to contact support, and they suggested I contact support via their [cannot sign in form](https://support.github.com/contact/cannot_sign_in). You cannot access this form unless you're logged out, of course. You will also be unable to reply to this support ticket; you will simply have to make a brand new one.

### Support Refuses To Comply With Most Requests Pertaining To User Data and User Accounts

GitHub support willy comply with certain requests. For example I wanted my billing information removed from my account and they complied. But when I requested they provide a copy of any user data they have gathered (if they even do collect any), they refused to comply with my request and instead told me to go to my account settings and go from there. But remember, I cannot do any of that without enabling 2FA. Furthermore, support went on to clarify that if you wish to do things such as delete your account, delete any repository data, etc, they will refuse to comply. **This can potentially be a violation of European and Californian consumer protection laws, as well as GitHub's own privacy policy**.

Lastly I want to end with this:
### Stop using Copilot. Use Codeium instead

EDIT:

Just got an email from support. Turns out you can reply to tickets via email (I thought you couldn't because they didn't respond after several days of me doing so).

Support got back to me and extended my 2FA requirement by 72 hours so that I can sign in and access any data on my account. I was able to get a copy of my data fortunately.

**Comment by VIPLightning** [comment]
This is Microsoft's doing and 2FA adds potentially greater security risk. That's pretty much what's happened to 100 million epic games users that were hacked by the Russians. But I digress. I'd switch and migrate all my work to [Codeberg](https://codeberg.org/) or other git websites before December 15 at 0:00 (UTC) arrives and shit hits the fan. Because when it happens, Microsoft pretty much made this website a high valuable target to hit and Microsoft's gonna do what's only in their best interest and not for consumers or developers.

**Comment by VIPLightning** [comment]
Btw Gitlab also enforces 2FA. Use Codeberg or Sourceforge

**Comment by MasterCATZ** [comment]
hate 2FA , just another way to be permanently locked out of an account when mobile phone is stolen or dies
